cves/kernel/CVE-2022-1975.yml

```diff
@@ -322,8 +333,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    Within the commit, it mentions being reviewed and signed off
+    by additional people.
```

Comment:
I think mentioning the people who signed off could be helpful here.

Reply:
Added the people.